examples/mobile/UIComponents/components/etc/themes/colors.html

```diff
@@ -22,7 +22,7 @@ <h1>
 			</h1>
 		</div>
 		<div class="ui-content">
-			<ul class="ui-listview" style="color: var(--color-white);">
+			<ul class="ui-listview ui-content-area" style="color: var(--color-white);">
```

Comment:
Why is the style code inline?
It is better to go towards css.

Reply:
It's sample for color value checking.
So I think it don't need to go towards css.

Reply:
ok. I will consider it next PR.

Reply:
I think moving styles to file is not scope of this change

Reply:
Yes. I'll think it in next time.